style.css

```diff
@@ -0,0 +1,313 @@
+    align-items: center;
+    flex-direction: column;
+}
+
+.Cards_Container {
+    height: 34.563rem;
+    border: 1px solid #C9C7FF;
+    display: flex;
+    border-radius: 0.5rem;
+    margin-top: 5.688rem;
+
+}
+
+.plan {
+    width: 25rem;
+    margin-top: 1.625rem;
+    border-right: 1px solid #C9C7FF;
+}
+
+
+.plan_data{
+    width: 378px;
+    margin-left: 32px;
+}
+.uper{
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+}
+.plan_name{
+    gap: 16px;
+    display: flex;
+    align-items: center;
+}
+.plan_name p{
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 22.8px;
+    color: #000000;
+}
+
+.plan h1 {
+    margin-top: 19px;
+    font-weight: 700;
+    font-size: 64px;
+    line-height: 60.8px;
+    color: #7C5BFF;
+}
+.uper h2 {
+    margin-top: 3rem;
+    font-size: 24px;
+    line-height: 22.8px;
+}
+
+.uper > p{
+    margin-top: 19px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 15.2px;
+    color: #6F6C90;
+
+}
+.plan button {
+    border: 1px solid #7C5BFF;
+    color: #7C5BFF;
+    background-color: transparent;
+    border-radius: 0.25rem;
+    margin-top: 27px;
+    padding: 7px 14px 7px 14px;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 16.71px;
+}
+
+.plan button:hover {
+    color: white;
+    background-color: #7C5BFF;
+    transition: 0.3s;
+}
+
+.info{
+    margin-top: 39px;
+}
+span{
+    font-weight: bold;
+    color: #000000;
+}
+
+.list{
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 19.09px;
+    margin-top: 8.08px;
+    display: flex;
+    align-items: center;
+    color: #6F6C90;
+
+}
+.tick{
+    width: 20px;
+    height: 20px;
+    border: 1px solid blue;
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-right: 12px;
+}
+.All_users{
+    margin-top: 127px;
+}
+.users {
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    margin-top: 13px;
+    width: 80vw;
+}
+
+.grid-item {
+    font-size: 14px;
+    font-weight: 600;
+    line-height: 20px;
+    padding: 7px;
+    gap: 6px;
+    display: flex;
+    align-items: center;
+}
+
+.grid-item:nth-child(-n+6) {
+    background-color: #F6F8FA;
+    font-weight: 600;
+    color:#30313D;
+    font-size: 14px;
+    line-height: 20px;
+}
+.grid-item:nth-child(-n+6):before {
+    content: "";
+    background-image: url("data:image/svg+xml,%3Csvg width='20' height='20' viewBox='0 0 20 20' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M10 9C11.6569 9 13 7.65685 13 6C13 4.34315 11.6569 3 10 3C8.34315 3 7 4.34315 7 6C7 7.65685 8.34315 9 10 9Z' fill='%2330313D'/%3E%3Cpath d='M6.49999 17.0002C5.86019 17.0002 5.22039 16.7561 4.73223 16.268C3.75592 15.2916 3.75592 13.7087 4.73223 12.7324C5.32149 12.1549 7.2 11 10 11C12.8 11 14.6785 12.1549 15.2678 12.7324C16.2441 13.7087 16.2441 15.2916 15.2678 16.268C14.7796 16.7561 14.1398 17.0002 13.5 17.0002H6.49999Z' fill='%2330313D'/%3E%3C/svg%3E%0A");
```

Comment:
Just to not, dont change it for now, 
use before after for placing icon only in listing, main purpose is re-usabillity